packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/workflow/WorkflowAnnotation.java

```diff
@@ -0,0 +1,22 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.workflow;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import java.io.Serializable;
+import java.util.List;
+import java.util.UUID;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.Accessors;
+
+@Data
+@Accessors(chain = true)
+public class WorkflowAnnotation implements Serializable {
+
+	private UUID id;
```

Comment:
We may want to store width and height

Reply:
I am thinking the UI should determine this,

Reply:
Fine by me, simple is good

Reply:
Then why the `textSize` if UI should determine design?

Reply:
Jamie wants to have small/medium/large text sizes as determined by the modeller.